src/components/ConfirmMessage/Message.tsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react'
+import React, { FC } from 'react'
 import { css, keyframes } from '@emotion/core'
 import styled from '@emotion/styled'
 import Markdown from 'react-markdown'
 import Link from '../Link'
 import { bpMaxSM } from '../../lib/breakpoints'
+import { Maybe } from '../../types/generated'
 
-export default ({
+type MessagePropsType = {
```

Comment:
for all components I created `PropsType`

Reply:
Wow, that's huge! thank you!